### PR TITLE
Psl 197 convert pastel gtest

### PR DESCRIPTION
**Pull request by tanlemin91**

### CODE REVIEW

File: src/gtest/test_dbwrapper.cpp [hunk 1]
```diff
@@ -1,236 +1,230 @@
...
+{
+    path ph = temp_directory_path() / unique_path();
+    CDBWrapper dbw(ph, (1 << 20), true, false);
+    for (int x=0x00; x<256; ++x) {
+        uint8_t key = x;
+        uint32_t value = x*x;
+        EXPECT_TRUE(dbw.Write(key, value));
+    }
+
+    boost::scoped_ptr<CDBIterator> it(const_cast<CDBWrapper*>(&dbw)->NewIterator());
```
Comment (akobrin1) [inline src/gtest/test_dbwrapper.cpp:128]: please replace boost::scoped_ptr with std::unique_ptr

File: src/gtest/test_dbwrapper.cpp [hunk 2]
```diff
@@ -1,236 +1,230 @@
...
+            assert(n > 0 && n < sizeof(buf));
+            StringContentsSerializer key(buf);
+            for (int z = 0; z < y; z++)
+                key += key;
+            uint32_t value = x*x;
+            EXPECT_TRUE(dbw.Write(key, value));
+        }
+    }
+
+    boost::scoped_ptr<CDBIterator> it(const_cast<CDBWrapper*>(&dbw)->NewIterator());
```
Comment (akobrin1) [inline src/gtest/test_dbwrapper.cpp:202]: same

**Review by akobrin1 — CHANGES_REQUESTED** [review]

File: src/gtest/test_dbwrapper.cpp (hunk 1 above)
Comment (tanlemin91) [reply src/gtest/test_dbwrapper.cpp]: thanks, fixed,

File: src/gtest/test_dbwrapper.cpp (hunk 2 above)
Comment (tanlemin91) [reply src/gtest/test_dbwrapper.cpp]: thanks, fixed,

**Review by akobrin1 — APPROVED** [review]

**Pull request merged**